hudi-client/src/test/java/org/apache/hudi/TestAsyncCompaction.java

```diff
@@ -106,17 +105,15 @@ public void testRollbackForInflightCompaction() throws Exception {
 
       HoodieInstant pendingCompactionInstant =
           metaClient.getActiveTimeline().filterPendingCompactionTimeline().firstInstant().get();
-      assertTrue("Pending Compaction instant has expected instant time",
-          pendingCompactionInstant.getTimestamp().equals(compactionInstantTime));
-      assertTrue("Pending Compaction instant has expected state",
-          pendingCompactionInstant.getState().equals(State.REQUESTED));
+      assertEquals("Pending Compaction instant has expected instant time", pendingCompactionInstant.getTimestamp(),
+          compactionInstantTime);
+      assertEquals("Pending Compaction instant has expected state", pendingCompactionInstant.getState(), State.REQUESTED);
 
       moveCompactionFromRequestedToInflight(compactionInstantTime, client, cfg);
 
       // Reload and rollback inflight compaction
       metaClient = new HoodieTableMetaClient(jsc.hadoopConfiguration(), cfg.getBasePath());
       HoodieTable hoodieTable = HoodieTable.getHoodieTable(metaClient, cfg, jsc);
-      hoodieTable.rollback(jsc, compactionInstantTime, false);
```

Comment:
this call is redundant, hoodieTable.rollback(jsc, compactionInstantTime, false) gets called as the first line of code in client.rollbackInflightCompaction(
          new HoodieInstant(State.INFLIGHT, HoodieTimeline.COMPACTION_ACTION, compactionInstantTime), hoodieTable);